--- gsm-1.0-pl10/src/toast.c.includes	1996-07-02 16:32:55.000000000 +0200
+++ gsm-1.0-pl10/src/toast.c	2004-04-14 14:31:22.000000000 +0200
@@ -6,7 +6,12 @@
 
 /* $Header: /tmp_amd/presto/export/kbs/jutta/src/gsm/RCS/toast.c,v 1.8 1996/07/02 10:41:04 jutta Exp $ */
 
+#define _DEFAULT_SOURCE
 #include	"toast.h"
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 
 /*  toast -- lossy sound compression using the gsm library.
  */
